package.json

```diff
@@ -64,6 +64,7 @@
     "update-snapshots": "lage update-snapshots --verbose"
   },
   "devDependencies": {
+    "vrscreenshotdiff": "0.0.6",
```

Comment:
nit: can you move this to line 307 to maintain alphabetical order of these packages?

Reply:
Done